KeywordSearch/ivy.xml

```diff
@@ -20,7 +20,7 @@
     <dependencies>
         <!-- Solr -->
         <dependency conf="solr-libs->default" org="org.apache.solr" name="solr-cell" rev="4.9.1"/>
-        <dependency conf="solr-war->default" org="org.apache.solr" name="solr" rev="4.9.1"/> <!-- the war file -->
+        <dependency conf="solr-war->default" org="org.apache.solr" name="solr" rev="4.9.1" transitive="false" /> <!-- the war file -->
```

Comment:
Without it I was ending up with a war file that was not functional. The documentation for transitive isn't particularly enlightening ("true to resolve this dependency transitively, false otherwise").